src/Orleans/Streams/SimpleMessageStream/SimpleMessageStreamProvider.cs

```diff
@@ -46,7 +46,7 @@ public Task Init(string name, IProviderRuntime providerUtilitiesManager, IProvid
             this.Name = name;
             providerRuntime = (IStreamProviderRuntime) providerUtilitiesManager;
             string fireAndForgetDeliveryStr;
-            fireAndForgetDelivery = !config.Properties.TryGetValue(FIRE_AND_FORGET_DELIVERY, out fireAndForgetDeliveryStr) || Boolean.Parse(fireAndForgetDeliveryStr);
+            fireAndForgetDelivery = config.Properties.TryGetValue(FIRE_AND_FORGET_DELIVERY, out fireAndForgetDeliveryStr) ? Boolean.Parse(fireAndForgetDeliveryStr) : false;
```

Comment:
To my taste, it would be slightly simpler with
`config.Properties.TryGetValue(FIRE_AND_FORGET_DELIVERY, out fireAndForgetDeliveryStr) && Boolean.Parse(fireAndForgetDeliveryStr);`
but that's just an insignificant style preference.